tgui/packages/tgui/interfaces/ApcControl.tsx

```diff
@@ -1,4 +1,4 @@
-import { sortBy } from 'common/collections';
+import { sortBy } from 'es-toolkit/compat';
```

Comment:
can we use sortby from the standard `es-toolkit` import? is there a reason / difference in reaching for /compat with things that aren't listed as compat? [says here](https://es-toolkit.dev/compatibility.html) that this makes it slower

> It's important to note that es-toolkit/compat may have a slight performance impact and a larger bundle size compared to the original es-toolkit

I understand instances where it's only offered by compat, like filter or map